documentation/modules/deploying/proc-deploying-kafkaconnector.adoc

```diff
@@ -3,16 +3,26 @@
 // assembly-kafka-connect.adoc
 
 [id='proc-deploying-kafkaconnector-{context}']
-= Deploying a `KafkaConnector` resource to Kafka Connect
+= Deploying the example `KafkaConnector` resource
 
-This procedure describes how to deploy the example `KafkaConnector` to a Kafka Connect cluster.
+Strimzi includes an example `KafkaConnector` in `examples/connect/source-connector.yaml`. 
+This creates a basic `FileStreamSourceConnector` instance that sends each line of the Kafka license file (an example file source) to a single Kafka topic. 
 
-The example YAML will create a `FileStreamSourceConnector` to send each line of the license file to Kafka as a message in a topic named `my-topic`.
+You can create a `FileStreamSinkConnector` by copying the example file and then changing the class and configuration. 
```

Comment:
There isn't an example file for the sink connector, is there?

Reply:
No, sadly not. It's awkward to explain to the user to copy the example file.

Reply:
So just create an example file?

Reply:
I've added steps to create `examples/connect/sink-connector.yaml` and use `kubectl exec` to read the messages. Please can you take another look.

@tombentley